modules/virtual_environments/auto-venv/venv_helpers.nu

```diff
@@ -23,21 +23,21 @@ export def venv-is-active [] {
     '__auto_venv' in (overlay list)
 }
 
-export def 'venv-create' [] {
 # Creates a virtual environment under the current directory
-	let venv_path = (pwd)
-	let venv_name = ((pwd) | path split | last) 
-	python3 -m venv .venv --clear --prompt $venv_name
-	.venv/bin/python3 -m pip install -U pip wheel setuptools
+export def 'venv-create' [] {
+	let venv_path = $env.PWD
+	let venv_name = ($env.PWD | path basename) 
+
+	let sys_posix = ['darwin', 'linux', 'unix', 'posix', 'gnu']
+	let is_posix = (((sys).host.name | str downcase) in $sys_posix)
+	let python_name = if $is_posix {'python3'} else {'py.exe'}
```

Comment:
you might be able to just do
```
let python = if $nu.os-info.name == "windows" { "py.exe" } else { "python3" }
```
here :smirk:

Reply:
Stackoverflow said `py.exe` but I have no idea. Someone with access to Windows 10/11 may be able to help here.

Reply:
Adopted `run-external`. Merge?